integrationTests/factory/componentsHelper.go

```diff
@@ -67,6 +67,8 @@ func CreateDefaultConfig() *config.Configs {
 	configs.RoundConfig = roundConfig
 	configs.FlagsConfig = &config.ContextFlagsConfig{
 		WorkingDir: "workingDir",
+		DbDir:      "workingDir",
```

Comment:
Not sure, should we use separate folders for tests, as well, instead?

Reply:
changed to use separate folders, not sure if it has a relevant impact in the tests though